monetdbe/_cffi/internal.py

```diff
@@ -268,7 +257,7 @@ def append(self, table: str, data: Mapping[str, np.ndarray], schema: str = 'sys'
     def prepare(self, query: str) -> monetdbe_statement:
         self._switch()
         stmt = ffi.new("monetdbe_statement **")
-        check_error(lib.monetdbe_prepare(self._monetdbe_database, str(query).encode(), stmt))
+        check_error(lib.monetdbe_prepare(self._monetdbe_database, str(query).encode(), stmt, ffi.NULL))
```

Comment:
how does this work with Jul2021?

Reply:
On Jul2021 there is a method override for the `prepare` at the end of internal.py. It is a bit ugly but the most concise and direct approach.

See

https://github.com/MonetDBSolutions/MonetDBe-Python/blob/16561d947406c01c6a9bf51f89e6b9540ed27796/monetdbe/_cffi/internal.py#L290-L298